controllers/rack.go

```diff
@@ -620,6 +620,12 @@ func (r *SingleClusterReconciler) scaleDownRack(
 		}
 	}
 
+	// set migrate-fill-delay to 0 across all nodes of cluster to scale down fast
+	// temporarily add the pod to be deleted in ignorablePods slice to avoid setting migrate-fill-delay to pod when pod is not ready
+	if res := r.setMigrateFillDelay(r.getClientPolicy(), rackState.Rack, false, append(ignorablePods, *pod)); !res.isSuccess {
```

Comment:
is there any reason of using r.getClientPolicy() while clearing migrate-fill-delay and r.getClientPolicyFromSpec() while setting migrate-fill-delay back to original value?

Reply:
I think it is because of reconcileAccessControl() being called in later point of time, nevermind.

Reply:
For the first time we need user from the spec which was created with sys-admin role because at this point AccessControl is not updated in the status. Later when Scaledown is triggered, we use AccessControl from the status to use the already created user as new AccessControl is yet to be reconciled